### PR TITLE
U net enhancements

### PR DESCRIPTION
**Pull request by kbali1297**
Added option to use residual blocks and swish activation function for U_net networks in torch, tensorflow and jax frameworks.

### CODE REVIEW

**Comment by holl-** [comment]
I've set the target branch to `2.2-develop`. Could you merge the latest changes into your branch? There should be no conflicts.

**Comment by kbali1297** [comment]
I've removed the conflicts. The request can now be merged.

File: tests/commit/Unit_Test_U_net.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                print("Tracing loss_function")
+                assert isinstance(x, math.Tensor)
+                pred = math.native_call(net, x)
+                return math.l2_loss(pred)
+
+            for i in range(2):
+                lib.update_weights(net, optimizer, loss_function, math.random_uniform(math.batch(batch=10), math.spatial(x=8, y=8)))
+
+
+test_obj = TestNetworks()
```
Comment (holl-) [inline tests/commit/Unit_Test_U_net.py:62]: These lines are not necessary. The tests will be automatically run by `pytest`

**Review by holl- — COMMENTED** [review]
Could you merge the functions from Unit_Test_U_net.py into `tests/commit/test_networks.py` and remove the final lines?

File: tests/commit/Unit_Test_U_net.py (hunk 1 above)
Comment (kbali1297) [reply tests/commit/Unit_Test_U_net.py:62]: I merged the functions from Unit_Test_U_net.py to existing test_networks.py file. I hope I didn't break anything.

Comment (holl-) [reply tests/commit/Unit_Test_U_net.py:62]: Okay, then `Unit_Test_U_net.py` can now be removed, right?

Comment (kbali1297) [reply tests/commit/Unit_Test_U_net.py]: Yes. I have removed it from the commit/tests now.

**Pull request merged**